config/ci_configs/gpu_slabplanet_atmos_diags.yml

```diff
@@ -0,0 +1,22 @@
+anim: false
```

Comment:
This config is copied from `config/ci_configs/slabplanet_atmos_diags.yml`. Once we remove `job_id` from inside the config file, we won't need to have both the CPU and GPU versions (though we'll have to make sure we always use `anim: false` for GPU runs)

Reply:
We can always dispatch the `plot_anim` on device and do nothing for GPU.

Reply:
Good idea!